domain/tests/src/Functional/DomainDrushCommandsTest.txt

```diff
@@ -0,0 +1,105 @@
+<?php
+
+namespace Drupal\Tests\domain\Functional;
+
+//use Drupal\Tests\domain\Functional\DomainCommandTestBase;
+
+/**
+ * Tests the domain record creation API.
+ *
+ * @group domain
+ */
+class DomainDrushCommandsTest extends DomainCommandTestBase {
+
+  /**
+   * Show some info about the domain module environment.
+   */
+  public function testInfoDomains() {
+    if ($this->isWindows()) {
+      $this->markTestSkipped('Site-set not currently available on Windows.');
+    }
+    $this->setUpDrupal(1, true);
+
+    // Assure that a pending post-update is reported.
```

Comment:
This whole section doesn't mean anything to me, as the comment doesn't match the code.

It's also not clear how 'domain_access_entities' is getting set.

Reply:
Now I have looked more carefully, I think the tests are showing an excess of cut-n-pasting. A lot of it is currently rubbish, so I will delete the files. I don't think they are adding anything useful.